recipes/libarchive/all/patches/0003-3.7.1-cmake.patch

```diff
@@ -56,3 +56,36 @@ index 6849ce40..8d5b6018 100644
    FIND_PACKAGE(OpenSSL)
    IF(OPENSSL_FOUND)
      SET(HAVE_LIBCRYPTO 1)
+diff --git a/libarchive/CMakeLists.txt b/libarchive/CMakeLists.txt
```

Comment:
Please push this change to the upstream via a PR. I still prefer filtering files into recipes than starting to maintain new patches.

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/sources_and_patches.md#policy-on-patches

Reply:
That's not possible to push this change. static lib must be built when you build executables (we disable these targets).